chainguard-dev/edu
# turn off emojis

## Type of change
fix #527 - turn off emoji rendering

### What should this PR do?
This turns off Hugo's [emoji rendering](https://gohugo.io/getting-started/configuration/#enableemoji) engine.

### Why are we making this change?
It messes up pre-rendered code blocks if an emoji byte code makes it into an example.

### What are the acceptance criteria? 
Pages should render normally.

### How should this PR be tested?
Check example pages like https://edu.chainguard.dev/open-source/sigstore/fulcio/how-to-inspect-and-verify-fulcio-certificates/ to ensure an emoji hasn't made it back in. Or, checkout any commit before `3fffa4f70c417944bcbf797972ceb5c84b26868a`, then edit the config to turn off the setting, and check the fulcio page (I did this and the issue in #524 goes away).
